app/Services/WebsiteUpdater/WebsiteUpdaterThroughGithubActions.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services\WebsiteUpdater;
+
+use GuzzleHttp\ClientInterface;
+use RuntimeException;
+
+final class WebsiteUpdaterThroughGithubActions implements WebsiteUpdater
+{
+    public function __construct(
+        private ClientInterface $client,
```

Comment:
Ik vind het een beetje lastig dat de `base_uri` wordt meegegeven in de service provider. Daardoor, als je alleen naar deze class kijkt, weet je niet er een base uri is en voelt het een beetje magisch

Reply:
goeie ik maak er een constant van